app/src/components/pages/Register.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from 'react';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Box from '@mui/material/Box';
+import { Grid, Typography } from '@mui/material';
+
+interface RegisterForm {
+  email: string;
+  username: string;
+  password: string;
+  confirmPassword: string;
+}
+
+function Register() {
+  const [hasErrors, setHasErrors] = useState<boolean>(false);
+  const [formData, setFormData] = useState<RegisterForm>({
+    email: '',
+    username: '',
+    password: '',
+    confirmPassword: ''
+  });
+
+  const createAccount = (e: { preventDefault: () => void }) => {
+    e.preventDefault();
+    console.log('Creating account: ', formData);

```

Comment:
Should probably drop this line, we don't want to print plaintext passwords anywhere